public/images/icons/animation/icon_lbh_spinner.svg

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Are you sure that we need spinner duplication in `public` folder? WE already have this jsx in `Loading`